README.md

```diff
@@ -28,6 +28,7 @@ The conference dataset hosted in this repository tracks the following informatio
 | [LambdaConf](https://lambdaconf.us) | Ziverge | Yes | Yes | Partial | Never |
 | [ZIO World](https://zioworld.com) | Ziverge | Yes | Yes | No | Never |
 | [ScalarConf](https://www.scalar-conf.com) | Software Mill | No | Yes | No | Never | 
+| [PyCon](https://pycon.org) | PyCon Board Committee | No | No | Yes | Partial |
```

Comment:
It seems highly unlikely that PyCon would reimburse expenses, but not provide speakers with a free ticket. Can you confirm this?

Reply:
Yes, I can confirm this. All speakers have to buy their own tickets, some (e.g. Keynote speakers) are compensated.

Reply:
Finally, did they say they would not sign SEA if explicitly requested to do so by an invited speaker, or is that just the assumption?

Reply:
They've said they would have to investigate when requested as they have never signed such a document before.

Reply:
Perhaps we should put `Pending` in the column until they initiate and finish investigation?

Reply:
That makes sense, I added that.